mriqc/interfaces/diffusion.py

```diff
@@ -545,3 +654,108 @@ def _extract_b0(in_file, b0_ixs, out_path=None):
 
 def _exp_func(t, A, K, C):
     return A * np.exp(K * t) + C
+
+
+def get_spike_mask(
```

Comment:
@arokem In the process of bringing this from the qc module.

I see the generation of this mask as a "preprocessing" step. The idea is to integrate it at an earlier step of the pipeline and feed it into the node that calculates IQMs. That way, we can define future IQMs using these masks without moving the calculation or re-doing it.